src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,16 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_GENDER = new Prefix("g/");
+    public static final Prefix PREFIX_BIRTHDAY = new Prefix("b/");
+    public static final Prefix PREFIX_IC_NUMBER = new Prefix("i/");
+    public static final Prefix PREFIX_DEPARTMENT = new Prefix("d/");
+
+    // Prefixes for Record
+    public static final Prefix PREFIX_OBSERVATION = new Prefix("o/");
+    public static final Prefix PREFIX_DIAGNOSIS = new Prefix("di/"); //d is taken
+    public static final Prefix PREFIX_TREATMENT_PLAN = new Prefix("tp/"); //t is taken
+    public static final Prefix PREFIX_REMARKS = new Prefix("r/");
+    public static final Prefix PREFIX_ACTIVE = new Prefix("ac/"); //a is taken
```

Comment:
I think these fields are no longer part of the Record class as discussed. Would be good to remove them!